File: .sdk/tm/csharp/utility/MakeContext.cs
// BluefinDecryptxP2pe SDK utility: makeContext.

namespace BluefinDecryptxP2peSdk.Util;

public static partial class SdkUtility
{
    internal static Context MakeContextUtil(Dictionary<string, object?>? ctxmap, Context? basectx)
    {
        return new Context(ctxmap, basectx);
    }
}

File: py/bluefindecryptxp2pe_sdk/utility/make_context.py
# BluefinDecryptxP2pe SDK utility: make_context

from bluefindecryptxp2pe_sdk.core.context import BluefinDecryptxP2peContext


def make_context_util(ctxmap, basectx):
    return BluefinDecryptxP2peContext(ctxmap, basectx)
